keras/layers/embeddings.py

```diff
@@ -51,16 +51,16 @@ class Embedding(Layer):
           If mask_zero is set to True, as a consequence, index 0 cannot be
           used in the vocabulary (input_dim should equal size of
           vocabulary + 1).
-      input_length: Length of input sequences, when it is constant.
+      input_length: Lengths of input sequences, when it is constant.
```

Comment:
If think it is confusing, especially for beginners (those who read the documentation) to have the docstring covered any other case than 2D. I would leave the docstring unchanged.